kt/godot-runtime/src/main/kotlin/godot/core/TransferContext.kt

```diff
@@ -19,9 +19,10 @@ object TransferContext {
         }
     }
 
-    private var inputStream  = object : InputStream() {
+    @ExperimentalUnsignedTypes
```

Comment:
Shouldn't we just add this as a compiler arg instead of annotations everywhere?
```kotlin
tasks.withType<KotlinCompile>().all {
    kotlinOptions.freeCompilerArgs += "-Xopt-in=kotlin.ExperimentalUnsignedTypes"
}
```

Reply:
I'm a way to late here, but here are my thoughts: This would enable the usage across the codebase, but ideally for experimental types we need to use them sparingly. Having to add annotation everytime is a good source of friction: `Do I really need this experimental feature?`